web/modules/custom/dpl_consumers/dpl_consumers.module

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
I think you should turn this into a drush cmd.
It is actualley quite [easy](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/228c256beaa05cedb2eca60242c12ae9b5eaede0/web/modules/custom/collation_fixer/src/Commands/CollationFixerCommands.php)

Can we maybe make the cmd a bit smaller, that can be convenient if we are going to use it a lot.
Eg.: `dev:go:graphql-credentials`

Reply:
Ah smart! I'll convert it into a drush cmd. 👍 

Yes, I wasn't sure if it was more important for the command names to be descriptive or "convenient" to use. But I agree, I'll change the cmd name.